2012-11-16  Pino Toscano  <toscano.pino@tiscali.it>

	* scripts/check-local-headers.sh: Ignore 'mach' headers.

--- a/scripts/check-local-headers.sh
+++ b/scripts/check-local-headers.sh
@@ -23,12 +23,13 @@ objpfx="$2"
 cd "$objpfx"
 
 # Search all dependency files for file names in the include directory.
-# There are a few system headers we are known to use.
+# There are a few system headers we are known to use,
+# and Mach kernel headers (mach).
 exec ${AWK} -v includedir="$includedir" '
 BEGIN {
   status = 0
   exclude = "^" includedir \
-    "/(.*-.*-.*/|)(asm[-/]|linux/|selinux/|gd|nss3/|sys/capability\\.h|libaudit\\.h)"
+    "/(.*-.*-.*/|)(asm[-/]|linux/|selinux/|mach/|gd|nss3/|sys/capability\\.h|libaudit\\.h)"
 }
 /^[^ ]/ && $1 ~ /.*:/ { obj = $1 }
 {
